src/app/state.rs

```diff
@@ -91,10 +92,10 @@ pub trait AppState {
                 #[cfg(feature="clipboard")]
                 {
                     let path = self.selected_path().to_string_lossy().to_string();
-                    match terminal_clipboard::set_string(path) {
+                    match clipboard::set_string(path) {
                         Ok(()) => AppStateCmdResult::Keep,
-                        Err(_) => AppStateCmdResult::DisplayError(
-                            "Clipboard error while copying path".to_string()
+                        Err( e ) => AppStateCmdResult::DisplayError(
+							format!( "{}", e ) 
```

Comment:
This prints an error that's "too techinical" I think, "Clipboard Error #NS<somethign somethign> failed to initialize". I am in favor of just saying: "Clipboard error" personally